deploy/helm/elastic-agent/values.schema.json

```
@@ -331,6 +331,36 @@
                 }
             }
         },
+        "auto_ops": {
+            "type": "object",
+            "description": "Configuration for AutoOps integration.",
+            "properties": {
+                "enabled": {
+                    "type": "boolean",
+                    "description": "Enable AutoOps integration."
+                },
+                "config": {
+                    "type": "string",
+                    "description": "AutoOps OTEL configuration file."
+                },
+                "envVars": {
+                    "type": "array",
+                    "items": {
+                        "type": "object"
+                    },
+                    "description": "AutoOps environment variables for configuration",
+                    "examples": [
+                        [
+                            {
+                                "name": "ENV_VAR_NAME",
+                                "value": "value"
+                            }
+                        ]
+                    ]
+
+                }
```

the properties here must derive from the properties you have introduced inside values.yaml
```suggestion
                "autoops_token": {
                    "type": "string",
                    "description": "..."
                },
                "autoops_es_url": {
                    "type": "string",
                    "description": "..."
                }
                ....
```